### PR TITLE
better width styling

### PR DESCRIPTION
**Pull request by zbtirrell**
https://central.tri.be/issues/101726

### CODE REVIEW

**Comment by zbtirrell** [comment]
For more clarity of how things function before and after this code: https://www.dropbox.com/s/14w3hcrnezz4c9d/screen-recording.mov?dl=0

File: src/resources/css/style.css [hunk 1]
```diff
@@ -3,6 +3,11 @@
  *
  * @package TribeEventsCalendarPro
  */
+
+.tribe_ext_sch_day_view #tribe-events-pg-template {
```
Comment (cliffordp) [inline src/resources/css/style.css:7]: This didn't take effect for me with Twenty Seventeen theme, but this CSS did:
`.tribe_ext_sch_day_view .tribe-events-day .tribe-events-loop { max-width: 100%; }`

Does that CSS solve the issue for you too on your theme?

**Comment by zbtirrell** [comment]
I'll test on 2017 :)

File: src/resources/css/style.css (hunk 1 above)
Comment (zbtirrell) [reply src/resources/css/style.css:7]: On 2017, it's not full width, but, it is the same width as List View (which is fine) and it doesn't get "stuck" on resize. So - I think the change as I presented it is good. 

Your recommended change would _also_ make it full width in 2017, but I'm not sure we need to do that.

Comment (cliffordp) [reply src/resources/css/style.css:7]: Gotcha. 👍

**Review by cliffordp — APPROVED** [review]

**Comment by cliffordp** [comment]
I'll merge, zip, and update the KB

**Pull request merged**